server/tests/parser_tests.py

```diff
@@ -34,8 +34,9 @@ def _test_case_parser(test_case):
             test_case['inputLines'] = '\n'.join(test_case['inputLines'])
 
     # Parse log entry text into one string
-    if 'logEntries' in test_case:
-        for log_entry in test_case['logEntries']:
+    if 'expectedResult' in test_case and \
```

Comment:
Just learned this today, we aren't supposed to use `\` for continuation lines. Just wrap the conditional in parentheses.